### PR TITLE
When I pay with OREID it should redirect transfers page

### PR DESCRIPTION
**Pull request by matjacdev515**
https://www.pivotaltracker.com/story/show/176797046

### CODE REVIEW

**Comment by Riddlerrr** [comment]
@Alexey1100 Do you know, OREID still doesn't return `state` param on an error?

**Comment by matjacdev515** [comment]
@Riddlerrr 
```
{"error_code"=>"sign_transaction_rejected", "error_message"=>"ChainErrorType:MiscTransactionError - Chain Transaction failure: 
  [{\"status\":400,\"response\":\"1\"},{\"req\":\"2\",\"header\":\"3\",\"status\":400,\"text\":\"4\"},
    {\"method\":\"5\",\"url\":\"6\",\"data\":\"7\",\"headers\":\"8\"},{\"content-type\":\"9\",\"content-length\":\"10\",
    \"connection\":\"11\",\"date\":\"12\",\"x-amzn-requestid\":\"13\",\"access-control-allow-origin\":\"14\",\"x-amz-apigw-id\":\"15\",
    \"x-amzn-trace-id\":\"16\",\"x-cache\":\"17\",\"via\":\"18\",\"x-amz-cf-pop\":\"19\",\"x-amz-cf-id\":\"20\"},\
    "{\\\"message\\\":\\\"TransactionPool.Remember: transaction 236QBROU6BVRR5JFTMTCUEKIZ37EDRI5PDVIATJWJNFLXUEPAZFA: 
    account N37KTWXPIEAJHEXOCPOIYDVKWKH64PTCLJCPSNMMDGVUBH6AGQ3AP7T2JQ balance 200000 below min 314000 (0 assets)\\\"}\\n\",\"POST\"
    ,\"https://testnet-algorand.api.purestake.io/ps2/v2/transactions\",
    {\"type\":\"21\",\"data\":\"22\"},{\"x-api-key\":\"23\",\"content-type\":\"24\"},\"application/json\",\"216\",\"close\",
    \"Mon, 08 Feb 2021 12:42:51 GMT\",\"57be1513-092c-4d12-ad99-c94f0854afa5\",\"*\",\"abS34FgBYosFuyw=\",
    \"Root=1-602131cb-7363d5b318d720566fc36425\",\"Error from cloudfront\",
    \"1.1 519f948816b5a5f20661b64827cc02b1.cloudfront.net (CloudFront)\",\"ORD51-C1\",\"2ij_kejji0_xtBQd4t_3caLcPDAG9KyRT63DIbaMNP8bXrbfM2d20Q==\"
    ,\"Buffer\",[130,163,115,105,103,196,64,82,127,73,225,211,4,172,236,127,232,164,235,63,22,67,107,96,147,147,126,195,117,241,122,250,99,210,11,6,22,233,36,185,24,58,198,232,159,164,164,78,93,107,220,198,251,31,248,93,137,32,188,218,100,81,247,253,30,231,130,111,69,78,0,163,116,120,110,137,164,97,112,97,110,1,164,97,112,105,100,206,0,202,77,132,163,102,101,101,205,3,232,162,102,118,206,0,186,123,152,163,103,101,110,172,116,101,115,116,110,101,116,45,118,49,46,48,162,103,104,196,32,72,99,181,24,164,179,200,78,200,16,242,45,79,16,129,203,15,113,240,89,167,172,32,222,198,47,127,112,229,9,58,34,162,108,118,206,0,186,127,128,163,115,110,100,196,32,110,254,169,218,239,65,0,147,146,238,19,220,140,14,170,178,143,238,62,98,90,68,249,53,140,25,171,64,159,192,52,54,164,116,121,112,101,164,97,112,112,108],
    \"MsmR0dy1P28Ztt79sLBcs964vplMD69qGOMZbJ84\",\"application/x-binary\"] on chain network algo_test for user email|6021311f2c673038a8b18506 of app t_504dfe37557746919cdf75c317f08ac2.
     Error from chain: [{\"originalError\":\"1\",\"errorType\":\"2\"},{\"status\":400,\"response\":\"3\"},\"MiscTransactionError\",
      {\"req\":\"4\",\"header\":\"5\",\"status\":400,\"text\":\"6\"},{\"method\":\"7\",\"url\":\"8\",\"data\":\"9\",\"headers\":\"10\"},
      {\"content-type\":\"11\",\"content-length\":\"12\",\"connection\":\"13\",\"date\":\"14\",\"x-amzn-requestid\":\"15\",
        \"access-control-allow-origin\":\"16\",\"x-amz-apigw-id\":\"17\",\"x-amzn-trace-id\":\"18\",\"x-cache\":\"19\",\"via\":\"20\",
        \"x-amz-cf-pop\":\"21\",\"x-amz-cf-id\":\"22\"},\"{\\\"message\\\":\\\"TransactionPool.Remember: transaction 236QBROU6BVRR5JFTMTCUEKIZ37EDRI5PDVIATJWJNFLXUEPAZFA: account N37KTWXPIEAJHEXOCPOIYDVKWKH64PTCLJCPSNMMDGVUBH6AGQ3AP7T2JQ balance 200000 below min 314000 (0 assets)\\\"}\\n\",
        \"POST\",\"https://testnet-algorand.api.purestake.io/ps2/v2/transactions\",{\"type\":\"23\",\"data\":\"24\"},{\"x-api-key\":\"25\"
        ,\"content-type\":\"26\"},\"application/json\",\"216\",\"close\",\"Mon, 08 Feb 2021 12:42:51 GMT\",
        \"57be1513-092c-4d12-ad99-c94f0854afa5\",\"*\",\"abS34FgBYosFuyw=\",\"Root=1-602131cb-7363d5b318d720566fc36425\",
        \"Error from cloudfront\",\"1.1 519f948816b5a5f20661b64827cc02b1.cloudfront.net (CloudFront)\",\"ORD51-C1\",
        \"2ij_kejji0_xtBQd4t_3caLcPDAG9KyRT63DIbaMNP8bXrbfM2d20Q==\",\"Buffer\",[130,163,115,105,103,196,64,82,127,73,225,211,4,172,236,127,232,164,235,63,22,67,107,96,147,147,126,195,117,241,122,250,99,210,11,6,22,233,36,185,24,58,198,232,159,164,164,78,93,107,220,198,251,31,248,93,137,32,188,218,100,81,247,253,30,231,130,111,69,78,0,163,116,120,110,137,164,97,112,97,110,1,164,97,112,105,100,206,0,202,77,132,163,102,101,101,205,3,232,162,102,118,206,0,186,123,152,163,103,101,110,172,116,101,115,116,110,101,116,45,118,49,46,48,162,103,104,196,32,72,99,181,24,164,179,200,78,200,16,242,45,79,16,129,203,15,113,240,89,167,172,32,222,198,47,127,112,229,9,58,34,162,108,118,206,0,186,127,128,163,115,110,100,196,32,110,254,169,218,239,65,0,147,146,238,19,220,140,14,170,178,143,238,62,98,90,68,249,53,140,25,171,64,159,192,52,54,164,116,121,112,101,164,97,112,112,108],
        \"MsmR0dy1P28Ztt79sLBcs964vplMD69qGOMZbJ84\",\"application/x-binary\"]", "process_id"=>"98d6c3d0dbdf"}
```

**Comment by matjacdev515** [comment]
Not sure if this will be used somewhere else, not on transfers page. Maybe I should store request referer instead?

**Review by aquabu — APPROVED** [review]

File: app/controllers/sign/ore_id_controller.rb [hunk 1]
```diff
@@ -11,13 +11,22 @@ def new
       source: current_account.address_for_blockchain(transfer.token._blockchain)
     )
 
+    session[:project_id] = transfer.award_type.project_id
```
Comment (Alexey1100) [inline app/controllers/sign/ore_id_controller.rb:14]: @matjacdev515 I've changed this action in #1772 so I'll address the update there 🙌

**Pull request merged**